src/components/AppNavigation/Settings.vue

```diff
@@ -256,8 +255,9 @@ export default {
 		},
 		defaultReminderOptions() {
 			const defaultAlarms = getDefaultAlarms().map(seconds => {
+				const label = seconds === 0 ? t('calendar', "At event start") : moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize()
```

Comment:
```suggestion
				const label = seconds === 0 ? t('calendar', 'At event start') : moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize()
```

CI doesn't like that :see_no_evil: 

If you could change that then we can merge - thanks so much!